routes/artist.js

```diff
@@ -0,0 +1,36 @@
+require('dotenv').config();
+
+const express = require('express');
+const app = express();
+const router = express.Router();
+const SpotifyWebApi = require('spotify-web-api-node');
+const hbs = require('hbs');
+
+app.set('view engine', 'hbs');
+app.set('views', __dirname + '/views');
+app.use(express.static(__dirname + '/public'));
+
+app.use(express.urlencoded({ extended: false }));
+app.use(express.json());
+
+const spotifyApi = new SpotifyWebApi({
+    clientId: process.env.CLIENT_ID,
+    clientSecret: process.env.CLIENT_SECRET
+});
+
+// Retrieve an access token
+spotifyApi
+  .clientCredentialsGrant()
+  .then(data => spotifyApi.setAccessToken(data.body['access_token']))
+  .catch(error => console.log('Something went wrong when retrieving an access token', error));
+
+router.get("/artist-search", (req, res) =>{
```

Comment:
This is a cleaner way to write the same route: https://prnt.sc/132pq7q. 
Same on the next ones.

And just for your knowledge, this would be the way to write the same route using _async_ _await_: https://prnt.sc/132pxw9
You can choose the method you prefer!